include/zenoh_commons.h

```diff
@@ -730,7 +773,7 @@ ZENOHC_API extern const char *Z_CONFIG_MULTICAST_INTERFACE_KEY;
 ZENOHC_API extern const char *Z_CONFIG_MULTICAST_IPV4_ADDRESS_KEY;
 ZENOHC_API extern const char *Z_CONFIG_SCOUTING_TIMEOUT_KEY;
 ZENOHC_API extern const char *Z_CONFIG_SCOUTING_DELAY_KEY;
-ZENOHC_API extern const char *Z_CONFIG_ADD_TIMESTAMP_KEY;
+ZENOHC_API extern const char *Z_CONFIG_TIMESTAMPING_ENABLED_KEY;
```

Comment:
This is technically a breaking change, is there a proper reason to change this name?